src/content/documentation/develop/developing-extensions-for-firefox-for-android-fennec.md

```diff
@@ -131,7 +131,9 @@ When setting `strict_min_version`, unless you’re targeting a specific version
 
 ## Install and run your extension in Firefox for Android
 
-In your extension, ensure that you've included an application ID using the [browser_specific_settings](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) key in the `manifest.json`:
+If you’re using web-ext, follow the [Testing in Firefox for Android](/documentation/develop/getting-started-with-web-ext#testing-in-firefox-for-android) instructions.
+
+If the extension is not signed, ensure that you've included an application ID using the [browser_specific_settings](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) key in the `manifest.json`:
```

Comment:
The process that follows is for non temporary installation, we may move clarify that by merging this with line 146, something like:

```
Otherwise, to install the extension non temporarily:
- ensure that you've included an application ID using the [browser_specific_settings](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) key in the `manifest.json`:
.... // fragment with the browser_specific_settings example
- [zip the content of your extension into an xpi file](/documentation/publish/package-your-extension) named to match the application ID, for example, `borderify@example.com.xpi`.

...
```

how that sounds to you?

Reply:
Fennec doesn't support `about:debugging`, which already implies that this is the process for generating an unsigned xpi file (the documentation writes that later). I'm minimizing the number of changes of the doc since this is basically an archived page (since Fennec isn't supported any more).

Reply:
I know it doesn't, but that doesn't mean that the page make it clear that "If the extension is not signed" means that.

anyway it is not such big deal to block on it.